site/book/03-packages/02-exploring-a-package.md

```diff
@@ -27,7 +27,7 @@ In addition, you can use a kpt function such as `search-replace` to run a query
 For example, to search for resources that have a field with path `spec.selector.tier`:
 
 ```shell
-$ kpt fn eval --image gcr.io/kpt-fn/search-replace:unstable wordpress -- 'by-path=spec.selector.tier'
+$ kpt fn eval wordpress --image gcr.io/kpt-fn/search-replace:unstable -- 'by-path=spec.selector.tier'
```

Comment:
gcr.io/kpt-fn/search-replace:v0.1

Reply:
The following commit has the changes that are suggested.